### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by ksapru**

### CODE REVIEW

File: sapru-krish.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Today's date is Feb 15th. The time is 4:11PM. This is for a class project for credit
```
Comment (sallyom) [inline sapru-krish.txt:1]: this should be in the file assignments/feb-15/sapru-krish.txt

**Review by sallyom — CHANGES_REQUESTED** [review]
joke.sh and hello-world.py should be in `assignments/sapru-krish/` folder

File: sapru-krish.txt (hunk 1 above)
Comment (ksapru) [reply sapru-krish.txt:1]: Renamed the file to sapru-krish using the "mv sapru sapru-krish" command

Comment (ksapru) [reply sapru-krish.txt:1]: Duplicate files deleted

**Pull request merged**